src/tests/JIT/Regression/JitBlue/Runtime_88091/Runtime_88091.cs

```diff
@@ -0,0 +1,152 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+
+public sealed class NamedSet
+{
+    public NamedSet(HashSet<string> set, string value)
+    {
+        Set = set;
+        Value = value;
+    }
+    public HashSet<string> Set;
+    public string Value;
+}
+
+class Program
+{
+    static int Main()
+    {
+        int result = 0;
+        try 
+        {
+            Problem(data);
+            result = 100;
+        }
+        catch (Exception e)
+        {
+            Console.WriteLine($"Failed: {e.Message}");
+            result = -1;
+        }
+        return result;
+    }
```

Comment:
Make this not `Main` (convention has been `TestEntryPoint` but it isn't semantically meaningful) and mark it with `[Fact]` (`using Xunit;`)

Optionally (not recommended here), instead make `Problem` a `[Theory]` and figure out `ClassData` for the input.

Optionally (recommend), make the entire method body `Problem(data)`. The infrastructure will handle exceptions and the 100 isn't required at that point (can just return void)

Reply:
Like so?

```C#
using XUnit;

class Runtime_88091
{
    [Fact] static int Test() => Problem(data);

```

Reply:
Yes, but (I missed this the first time) also make it (the method -and- class) public.

To verify type "88091" in the test tab of the azdo job for the PR and see if it finds the test.  It won't if it isn't public, doesn't have [Fact], etc.  There are a few checks in msbuild and the analyzers, but there are incomplete and I'm hoping to improve them so that you get direct feedback instead of the current situation.

Reply:
> The infrastructure will handle exceptions

As far as I've seen in #88006, it does mark the test as failed but it doesn't print the exception in the log in such case.

Reply:
I'll see if I can fix the exception reporting.

The test is actually running.  There seems to be something strange with the azdo test filter where it doesn't show up for a while.  I don't know if it is time-based or if it's based on earlier queries.  (I also didn't see it but then after looking for "JIT", "Regression", etc., it started showing up.  I also see it in the helix log.)

Reply:
Yeah I see it now. Thanks!

<img width="709" alt="image" src="https://github.com/dotnet/runtime/assets/10121823/7d085e2b-09cf-482c-9a7f-703751c77463">